airflow/www/templates/airflow/dag_details.html

```diff
@@ -25,11 +25,16 @@
     {{ super() }}
     <h2>{{ title }}</h2>
     <div>
-      {% for state, count in states %}
+      {% for state, count in states%}
```

Comment:
Should there be a space between `states` and `%}`?

Reply:
Whops yes.